src/Data/RouteData.ts

```diff
@@ -0,0 +1,90 @@
+export class RouteData {
+    public static getCategories() {
+        return RouteData.getRoutes().categories;
+    }
+
+    private static getRoutes() {
+        const routes = {
+            'name': 'Puzztool',
+            'categories': [
+                {
+                    'name': 'Ciphers - Scrambled Letters',

```

Comment:
Can you make these match the menu option names and move the description to a separate field? Once this land in planning to use the same data to generate the menus as well.

Reply:
Also for completeness can you add the root url (E.g. '/cipher'), it's used for menu highlighting.